main.py

```diff
@@ -563,9 +565,12 @@ def generate_new_readme(stats: str, readme: str):
         rdmd = decode_readme(contents.content)
         new_readme = generate_new_readme(stats=waka_stats, readme=rdmd)
         if commit_by_me.lower() in truthy:
-            committer = InputGitAuthor(username, email)
+            committer = InputGitAuthor(username or commit_username, email or commit_email)
         else:
-            committer = InputGitAuthor('readme-bot', '41898282+github-actions[bot]@users.noreply.github.com')
+            committer = InputGitAuthor(
+                commit_username or 'readme-bot',

```

Comment:
Why do you set the default value in action.yml?

Reply:
41898282 is GitHub action bot ID, if the user does not set commit_by_me so we can define a bot for commiting.

Reply:
and GitHub API now did not return email so I add field commit_email that we can provide our email.